todo-demo-app/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.application.name=todo-demo-dummy-app
+quarkus.http.root-path=/api
+quarkus.kubernetes.deployment-target=openshift
```

Comment:
You don' need the 3 lines above. If you use `io.quarkus:quarkus-openshift`, then `quarkus.kubernetes.deployment-target` is automatically set to `openshift`. The first 2 lines are just non-default configuration, but defaults should work just fine for you.